templates/bundle/resources/usr/local/bin/liferay_entrypoint.sh

```diff
@@ -83,4 +85,16 @@ function start_monitor_liferay_lifecycle {
 	fi
 }
 
+function generate_thread_dump_now {
+	if [[ -s "${LIFERAY_DOCKER_AUTO_THREAD_DUMP_SLEEP_FILE}" ]]
+	then
+		while [ -f "${LIFERAY_DOCKER_AUTO_THREAD_DUMP_SLEEP_FILE}" ]
+		do
+			/usr/local/bin/generate_thread_dump.sh
+			
+			sleep $(cat "${LIFERAY_DOCKER_AUTO_THREAD_DUMP_SLEEP_FILE}")
+		done &
```

Comment:
I'd rather have the process in a separate function than use the "while ... done &", it's not very readable this way.